pkg/plugins/secrets/k8s/store_test.go

```diff
@@ -295,7 +295,7 @@ var _ = Describe("KubernetesStore", func() {
 			Expect(err).To(MatchError(store.ErrorResourceNotFound(core_system.SecretType, name, "demo")))
 		})
 
-		It("should return an existing resource", func() {
+		It("should return an existing resource with explicit mesh label", func() {
```

Comment:
We don't use labels, so maybe better to say ` with explicit mesh set`?

Reply:
but for Secrets we actually do use labels

Reply:
Oh, right. Secrets are special ;)